### PR TITLE
Migration from deprecated Long constructor

### LINKED ISSUE #3287: Feat: Suggested migration from deprecated use of new Long(long)
We should migrate this to Long.valueOf(long) syntax instead everywhere in the repo

image

### PR DESCRIPTION
**Pull request by SoulPancake**
Fixes #3287

### CODE REVIEW

**Review by sazzad16 — CHANGES_REQUESTED** [review]
There are more instances of new Long. Please find and replace those.

Moreover, there are few commented codes in PipeliningTest. Please change those as well.

**Comment by SoulPancake** [comment]
@sazzad16 Oh, So I should modify the commented ones as well?
I was under the impression that it wouldn't matter.

**Comment by SoulPancake** [comment]
> There are more instances of new Long. Please find and replace those.
> 
> Moreover, there are few commented codes in PipeliningTest. Please change those as well.

I've made the above changes and re-requested for review.

**Review by sazzad16 — CHANGES_REQUESTED** [review]
image

File: src/test/java/redis/clients/jedis/commands/jedis/ObjectCommandsTest.java [hunk 1]
```diff
@@ -48,11 +48,11 @@ public void tearDown() throws Exception {
   public void objectRefcount() {
     jedis.lpush(key, "hello world");
     Long refcount = jedis.objectRefcount(key);
-    assertEquals(new Long(1), refcount);
+    assertEquals(Long.valueOf(1), refcount);
```
Comment (SoulPancake) [inline src/test/java/redis/clients/jedis/commands/jedis/ObjectCommandsTest.java:51]: @sazzad16 As you can see I have made those changes as well

File: src/test/java/redis/clients/jedis/commands/jedis/PublishSubscribeCommandsTest.java [hunk 2]
```diff
@@ -185,7 +185,7 @@ public void onPSubscribe(String pattern, int subscribedChannels) {
         if (count == 3) {
           Jedis otherJedis = createJedis();
           Long numPatterns = otherJedis.pubsubNumPat();
-          assertEquals(new Long(2l), numPatterns);
+          assertEquals(Long.valueOf(2l), numPatterns);
```
Comment (SoulPancake) [inline src/test/java/redis/clients/jedis/commands/jedis/PublishSubscribeCommandsTest.java:188]: @sazzad16 Reference :

**Comment by SoulPancake** [comment]
@sazzad16 Did you get a chance to review this?

**Review by sazzad16 — APPROVED** [review]

**Pull request merged**